src/utils/constants.js

```diff
@@ -1,5 +1,5 @@
 export const EventType = {
-  foodDrive: "FOODDRIVE",
+  foodDrive: "FoodDrive",
```

Comment:
We can live the value to all caps

Reply:
I ran into issues with querying - the type has to match the one defined in the schema

Reply:
Was the issue still there if the schema is in all caps too?